requirements/base

```diff
@@ -0,0 +1,4 @@
+-r xcffib
```

Comment:
Is this a working hack so that xcffib is always installed first?!

Reply:
yeah

Reply:
err, no, sorry, the hack is in tox.ini

Reply:
this is just for completeness, an in hopes someday it will just work

Reply:
my hopes were high but alas no, that doesn't work:

```
$ echo xcffib > xcffib
$ echo -e "-r xcffib\ncairocffi\n" > hack
$ pip install --no-cache-dir -r hack
Collecting cairocffi
  Downloading cairocffi-1.2.0.tar.gz (70 kB)
     |████████████████████████████████| 70 kB 1.4 MB/s 
Collecting cffi>=1.1.0
  Downloading cffi-1.14.4-cp37-cp37m-manylinux1_x86_64.whl (402 kB)
     |████████████████████████████████| 402 kB 2.2 MB/s 
Collecting xcffib
  Downloading xcffib-0.11.1.tar.gz (84 kB)
     |████████████████████████████████| 84 kB 3.7 MB/s 
Collecting pycparser
  Downloading pycparser-2.20-py2.py3-none-any.whl (112 kB)
     |████████████████████████████████| 112 kB 3.9 MB/s 
Collecting six
  Downloading six-1.15.0-py2.py3-none-any.whl (10 kB)
Building wheels for collected packages: cairocffi, xcffib
  Building wheel for cairocffi (setup.py) ... done
  Created wheel for cairocffi: filename=cairocffi-1.2.0-py3-none-any.whl size=89545 sha256=ff339b46df2ebf5df7355fef6763c1a6aaf8b9daf904c026d970125e2ec7a278
  Stored in directory: /tmp/pip-ephem-wheel-cache-uk3riam1/wheels/e2/ca/86/9db2824f203afe4bdf5aa6ead017f352fcc90e92f095b78871
  Building wheel for xcffib (setup.py) ... done
  Created wheel for xcffib: filename=xcffib-0.11.1-py3-none-any.whl size=95223 sha256=cadfd8f4d75dcae8b211f81f603f03c12b0e97409b0632fa3bbca6e8d30c6178
  Stored in directory: /tmp/pip-ephem-wheel-cache-uk3riam1/wheels/c6/55/fd/d2964bac524a60cb68d24b09d4cc845c47cb4e4e21f852c10a
Successfully built cairocffi xcffib
Installing collected packages: pycparser, six, cffi, xcffib, cairocffi
Successfully installed cairocffi-1.2.0 cffi-1.14.4 pycparser-2.20 six-1.15.0 xcffib-0.11.1
```

Reply:
sorry I was confused, I added comments above.

Reply:
then maybe we should just add xcffib in base and remove that file? I don't see a need for a separate file

Reply:
the only reason is to have the version requirement for xcffib in one place instead of two